doc/releases/release-notes-3.6.rst

```diff
@@ -488,14 +488,17 @@ Libraries / Subsystems
 
 * LoRa/LoRaWAN
 
+ * Added LoRaWAN remote multicast support with :kconfig:option:`CONFIG_LORAWAN_REMOTE_MULTICAST`
+   in preparation for OTA firmware upgrade support.
+
 * RTIO
 
 * ZBus
 
-  * Renamed :kconfig:option:`ZBUS_MSG_SUBSCRIBER_NET_BUF_DYNAMIC` and
-    :kconfig:option:`ZBUS_MSG_SUBSCRIBER_NET_BUF_STATIC`
-    with :kconfig:option:`ZBUS_MSG_SUBSCRIBER_BUF_ALLOC_DYNAMIC` and
-    :kconfig:option:`ZBUS_MSG_SUBSCRIBER_BUF_ALLOC_STATIC`
+  * Renamed :kconfig:option:`CONFIG_ZBUS_MSG_SUBSCRIBER_NET_BUF_DYNAMIC` and
+    :kconfig:option:`CONFIG_ZBUS_MSG_SUBSCRIBER_NET_BUF_STATIC`
+    with :kconfig:option:`CONFIG_ZBUS_MSG_SUBSCRIBER_BUF_ALLOC_DYNAMIC` and
+    :kconfig:option:`CONFIG_ZBUS_MSG_SUBSCRIBER_BUF_ALLOC_STATIC`
```

Comment:
The old values should be enclosed in double backticks.

Reply:
and this probably rather belongs to migration guide anyway, right?

Reply:
I was just fixing up incorrect usage of `:config:option:`, not adding this section.